### PR TITLE
Update azure_rm_servicebus related document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_servicebus related document.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_servicebus/azure_rm_servicebus_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_servicebus.py [hunk 1]
```diff
@@ -67,9 +65,11 @@
 '''
 RETURN = '''
 id:
-    description: Current state of the service bus.
+    description:
+        - Current state of the service bus.
     returned: success
     type: str
+    sample: "/subscriptions/xxx...xxx/resourceGroups/v-xisuRG/providers/Microsoft.ServiceBus/namespaces/nsb57dc9561"
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_servicebus.py]: v-xisuRG -> myResourceGroup. nsbxxx -> myServicebus

File: lib/ansible/modules/cloud/azure/azure_rm_servicebus_facts.py [hunk 2]
```diff
@@ -95,227 +95,269 @@
 '''
 RETURN = '''
 servicebuses:
-    description: List of servicebus dicts.
+    description:
+        - List of servicebus dicts.
     returned: always
-    type: list
+    type: complex
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_servicebus_facts.py:101]: list

Comment (Fred-sun) [reply lib/ansible/modules/cloud/azure/azure_rm_servicebus_facts.py:101]: if describe nested return values, set type: complex and repeat the elements above for each sub-field (Azure format [document's link](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html)).

**Review by samccann — APPROVED** [review]

**Pull request merged**